docs/contributing/blog-and-website-contributions.md

```diff
@@ -11,50 +11,36 @@ Here are some things to keep in mind when deciding where to contribute to Gatsby
-
-```md
----
-title: "Your Great Blog Post"
-date: YYYY-MM-DD
-author: Jamie Doe
-excerpt: "Here is a helpful excerpt or brief description of this blog post."
-tags:
-  - awesome
-  - post
----
-
-Your next great blog post awaits!
-
-Include images by creating a folder for your post and including
-Markdown and image files for easy linking.
-
-![awesome example](./image.jpg)
-```
+If you'd like to contribute a post to the Gatsby blog, please review the process and guidelines outlined below and submit your
+idea for the post to our [Gatsby blog proposal form](https://airtable.com/shr3449954866i3iF)
+
+### Blog proposal submission process
+1. Complete and submit the [Gatsby blog proposal form](https://airtable.com/shr3449954866i3iF).
+2. A Gatsby team member will review your proposal and let you know if the proposal has been accepted within the next week or so.
+    - __If the post is accepted:__ A Gatsby team member will work with you on a timeline for submitting and reviewing a draft of your blog post and set a tentative publishing date.
+    - __If the post is not accepted:__ We’ll let you know if there are any alternative offers we can make (e.g. offer to retweet if you publish the piece elsewhere, suggest submitting it as an addition to a Gatsby doc, etc.). We’ll also do our best to explain why your proposal was not accepted and encourage you to revise your proposal based on that feedback and resubmit. Please don’t be discouraged from submitting another post in the future!
+
+If you have any questions about the process or your submission, please email [marketing@gatsbyjs.com](mailto:marketing@gatsbyjs.com).
+
+### Content guidelines for submitting a blog post proposal
+As a Gatsby community member, you have unique insight into the ins and outs of learning Gatsby, building with Gatsby, and contributing to Gatsby’s open source community. Contributing to the Gatsby blog is a great way to share your experiences and insights. Here are some guidelines for what kind of content is and isn’t a good fit for the Gatsby blog.
+
+Things we’re looking for in Gatsby blog content:
+- Information to help others overcome challenges you’ve faced while working with Gatsby
+- Stories about how Gatsby helped you overcome different challenges on work and personal projects
+- Gatsby case studies
+- Showcasing a tool, fix, or other content you’ve contributed to Gatsby’s open source community
+- Showcasing a tool, fix, or other content someone else has contributed to Gatsby’s open source community
+- Clear and thoughtful explanations of technical details or complex concepts related to React, Graph QL, web and application development, open-source contribution, Gatsby core, and other Gatsby-adjacent subject matter
```

Comment:
Correcting the usage of `GraphQL` -- but I feel like this is too close to Reference Guides. Here's where I wrote about the distinction: https://www.gatsbyjs.org/contributing/docs-templates/ 
```suggestion
- Clear and thoughtful explanations of technical details or complex concepts related to React, GraphQL, web and application development, open-source contribution, Gatsby core, and other Gatsby-adjacent subject matter
```

Reply:
I think a blog post would maybe be a little more focused on a specific use case or approach to a challenge. And the overall content style would be more editorial/journalistic. It's something we're just going to have to figure out on a case-by-case basis, I think.